app/renderer/components/Inspector/HighlighterRects.js

```diff
@@ -1,20 +1,165 @@
 import React, { Component } from 'react';
 import HighlighterRect from './HighlighterRect';
+import HighlighterCentroid from './HighlighterCentroid';
 import B from 'bluebird';
-import { SCREENSHOT_INTERACTION_MODE } from './shared';
+import { SCREENSHOT_INTERACTION_MODE, parseCoordinates, RENDER_CENTROID_AS } from './shared';
 
 const {TAP, SWIPE, SELECT} = SCREENSHOT_INTERACTION_MODE;
+const {CENTROID, OVERLAP, EXPAND} = RENDER_CENTROID_AS;
+
+// Centroid positions are shifted based off screenshot width % for perfect alignment
+const POS_OFFSET = 0.03;
```

Comment:
Yup, thats a great point and I made changes to now ensure that we are not using any magic numbers and instead we are just relying on CSS in terms of making sure the distances between the overlapping centroids and the positioning of the centroids are more accurate, consistent and I don't think it will produce any bugs in the future as well now!